app/models/banner.rb

```diff
@@ -0,0 +1,10 @@
+class Banner < ActiveRecord::Base
+  scope :active, -> { where('expires_at > ?', DateTime.now) }
+
+  validates :message, presence: true
+  validates :expires_at, presence: true
+
+  def expires_at=(expires_at)
+    super(expires_at).try(:in_time_zone, 'Central Time (US & Canada)')
```

Comment:
Should be fine, but do you really care about the return value of `expires_at=`?

Reply:
No I don't care about the return value, but I think what that code does is *parse* the time as Central time because that's what it is. Otherwise, it thinks that it's in UTC which it isn't. @Dantemss

Reply:
What this code is doing is setting the expires_at to whatever time you pass in, then return that value as Central Time. I don't think the return part is needed.

Reply:
Maybe just remove this method, I don't think it's actually doing much.

Reply:
Ohh. Yeah. Done.